src/sentry/static/sentry/app/components/modals/addDashboardWidgetModal.tsx

```diff
@@ -370,7 +370,10 @@ class AddDashboardWidgetModal extends React.Component<Props, State> {
         </Body>
         <Footer>
           <ButtonBar gap={1}>
-            <Button external href="https://docs.sentry.io/product/dashboards/">
+            <Button
+              external
+              href="https://docs.sentry.io/product/dashboards/custom-dashboards/"
```

Comment:
Thoughts on going directly to the widget builder?
```suggestion
              href="https://docs.sentry.io/product/dashboards/custom-dashboards/#widget-builder"
```